### DISCUSSION TITLE
Rendering from an image sequence to a video file

### DISCUSSION THREAD

**Discussion by voganesyan** [inlife/nexrender#658] [Q&A]
Hello. Is it possible to convert an image sequence to a video file using nexrender? How should the `src` field of the `assets` object look like in this case?

**Comment by inlife** [comment]
Not an easy question; I would assume it would be easier to use FFmpeg to do that. Or if nexrender is a necessity, use pre-render action to do that.

**Reply by voganesyan** [reply]
@inlife thanks for your reply. Unfortunately, FFmpeg doesn't have the encoder I need (NotchLC). And I did't get to know how to use pre-render action. But it seems I've found another solution which doesn't require nexrender, but does require a little coding: replace some fields in a template project file (path to the image folder, frame size, etc) and pass it directly to aerender.exe.
